apps/website/blog/2023-10-11-frontend-plugin/index.mdx

```diff
@@ -0,0 +1,451 @@
+---
+title: Create and integrate Backstage Frontend plugin with Backend plugin
+authors: [sgahlot]
+tags: [Frontend, Plugins, Integrate, UI, Backend]
+description: Create a Backstage frontend plugin and integrate it with an existing backend plugin
+---
+
+# Creating Backstage Frontend plugin and integrate it with a Backend plugin
+
+## Introduction
+
+This blog will provide information on how we can create a Backstage Frontend plugin as well integrate it with an existing Backstage Backend plugin. The Backend plugin we will use in this article is mentioned in another [blog](https://janus-idp.io/blog/2023/08/23/creating-your-first-backstage-backend-plugin).
+
+We will make some [modifications to the backend plugin](#modify-backstage-backend) to wire it into the Backstage app and then move on to [create a new frontend plugin](#create-frontend-plugin). Once everything is in place and we have verified the frontend plugin integration with the backend plugin, we will also [analyze the code](#analyze-the-code) that makes it possible.
+
+### Prerequisites
+
+To start, please make sure that the following tasks have already been performed:
+
+- [Run Backstage Showcase locally](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/getting-started.md#running-locally-with-a-basic-configuration)
+- [Created your first Backstage Backend plugin](https://janus-idp.io/blog/2023/08/23/creating-your-first-backstage-backend-plugin)
+  - We will refer to this plugin as _backend plugin_
+
+---
+
+## Modify Backstage backend
+
+The backend plugin exposes `/system-info` endpoint that is only available using curl, and is not accessible through the Backstage app to any Frontend plugin. To expose and make use of this plugin in our app by wiring it into the Backstage backend, we will need to make some modifications to the Backstage backend.
+
+### Export the backend plugin router
+
+- Create a new file named `packages/backend/src/plugins/local-system-info.ts` with the following contents:
+
+  ```ts
```

Comment:
I just want to stay consistent with showing which files these code blocks should correspond to since I noticed you sometimes didn't include the name of the file in the code blocks and sometimes did.

Reply:
@Zaperex Good catch. Updating it now